Lib/test/test_pickletools.py

```diff
@@ -514,6 +519,172 @@ def test__all__(self):
         support.check__all__(self, pickletools, not_exported=not_exported)
 
 
+class CommandLineTest(unittest.TestCase):
+    def setUp(self):
+        self.filename = tempfile.mktemp()
```

Comment:
I think we have an os_helper functions for creating temp files without the needs to import tempfil.mktemp etc

Reply:
As fat i know we have `temp_dir` but don't have same function for files.
And i take this code from your tests `dis` cli :)

Reply:
I knew it... the code appeared quite familiar... well I got bitten! I was almost sure that we had one!!

Reply:
Otherwise, use `test.support.os_helper.TESTFN`?

Reply:
I can use it in `setUp`, but I also use temp files in other functions like `test_output_flag`, and when I use replace there I get `ValueError: pickle exhausted before STOP`

Reply:
Oh ok. Let's keep it as you wrote then.